public/styles/style.css

```diff
@@ -10058,6 +10057,11 @@ body {
   line-height: 5rem;
   align-items: center;
 }
+@media (max-width: 450px) {
+  .amplication-base .roadmap-heading {
+    line-height: 3rem;
```

Comment:
why do we need this?

Reply:
If we don't add this the 'line-height' will be '5 rem' for mobile view also